docs/source/release_notes/release-notes.rst

```diff
@@ -44,21 +44,25 @@ develop
     *    - Type
          - Change
 
-    *    - |new|
-         - A new config option `initializeAsync` was added to AtlasDbConfig. If set to true, it will allow a SnapshotTransactionManager to be created even if the KVS is not up.
-           Calling any method on the returned TransactionManager will throw a NotInitializedException until it gets initialized - this is, until the backing store becomes available.
-           The default value for the config  is `false` in order to preserve previous behavior.
+    *    - |new| |improved|
+         - AtlasDB now supports asynchronous initialization, where ``TransactionManagers.create()`` creates a SnapshotTransactionManager even when initialization fails, for instance because the KVS is not up yet.
+
+           To enable asynchronous initialization, a new config option ``initializeAsync`` was added to AtlasDbConfig.
+           If this option is set to true, `TransactionManagers.create()` first attempts to create a SnapshotTransactionManager synchronously, i.e., consistent with current behaviour.
```

Comment:
rst typo: single backtick used instead of double backtick (``) here. Should also wrap the class names `SnapshotTransactionManager` and `AtlasDbConfig` in double backticks.

Reply:
And `NotInitializedException`.

Reply:
And `false`.